projects/mci/test/test_mci_output.c

```diff
@@ -168,6 +174,15 @@ EEDriveOutput TEST_MOCK(drive_fsm_get_drive_state)() {
   return s_drive_state;
 }
 
+bool TEST_MOCK(drive_fsm_is_cruise)() {
+  return s_is_cruise;
+}
+
+// Mocking this function is somehow required to prevent a Segfault.
+bool TEST_MOCK(drive_fsm_toggle_cruise)() {
```

Comment:
If this function is not mocked, reading the current+velocity from line 681 of this file produces a SEGFAULT - will only occur for this test and no other tests. I hope this doesn't happen outside of a test setting?